src/browser/declarativewebutils.cpp

```diff
@@ -122,9 +122,11 @@ void DeclarativeWebUtils::updateWebEngineSettings()
     SailfishOS::WebEngine *webEngine = SailfishOS::WebEngine::instance();
 
     webEngineSettings->setPreference(QStringLiteral("general.useragent.updates.url"),
-                        QStringLiteral("https://browser.sailfishos.org/gecko/%APP_VERSION%/ua-update.json"));
-    webEngineSettings->setPreference(QStringLiteral("general.useragent.updates.interval"), QVariant(172800)); // every 2nd day
-    webEngineSettings->setPreference(QStringLiteral("general.useragent.updates.retry"), QVariant(86400)); // 1 day
+                        MGConfItem(QStringLiteral("/apps/sailfish-browser/settings/useragent_update_url")).value(QStringLiteral("https://browser.sailfishos.org/gecko/%APP_VERSION%/ua-update.json")));
```

Comment:
To make this line a bit shorter you could have 

```
static const auto defaultUserAgentUpdateUrl = QStringLiteral("https://browser.sailfishos.org/gecko/%APP_VERSION%/ua-update.json");
```
Some where around line 38 and use that defaultUserAgentUpdateUrl here as default value.
I'd not mind if you declare those dconf keys in similar fashion. There are some old QString that could QStringLiterals as well.